test/base/MockExportsCacheService.scala

```diff
@@ -16,15 +16,17 @@
 
 package base
 
-import org.mockito.ArgumentCaptor
+import org.mockito.{ArgumentCaptor, Mockito}
 import org.mockito.ArgumentMatchers.{any, anyString}
 import org.mockito.Mockito.{never, verify, when}
+import org.scalatest.{BeforeAndAfterEach, Suite}
 import org.scalatest.mockito.MockitoSugar
 import services.cache.{ExportsCacheModel, ExportsCacheModelBuilder, ExportsCacheService}
 
 import scala.concurrent.Future
 
-trait MockExportsCacheService extends MockitoSugar with ExportsCacheModelBuilder {
+trait MockExportsCacheService extends MockitoSugar with ExportsCacheModelBuilder with BeforeAndAfterEach {
+  self: Suite =>
```

Comment:
Why we need this Suite?

Reply:
BeforeAndAfterEach is posible to mix in only to `Suite`

Reply:
Cheers!

Reply:
It works in lots of test classes that don't have the `self: Suite =>`?